apps/Dataman/src/components/useLocation.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import * as Location from "expo-location";
+
+export const useLocation = () => {
+  const [errorMsg, setErrorMsg] = useState("");
+  const [locationServiceEnabled, setLocationServiceEnabled] = useState(false);

```

Comment:
for a boolean value prefer names such as `isLocationServiceEnabled`

Reply:
done